siliconcompiler/apps/sc_dashboard.py

```diff
@@ -40,7 +40,13 @@ def main():
                        'action': 'append',
                        'help': 'chip name - optional, path to chip manifest (json)',
                        'metavar': '<[manifest name, manifest path>',
-                       'sc_print': False}
+                       'sc_print': False},
+        "--configuration": {'type': str,
+                            'nargs': '+',
+                            'action': 'append',
+                            'help': 'path to chip configuration file (json)',
+                            'metavar': '<path to JSON>',
+                            'sc_print': False}
```

Comment:
it looks like only one can be used, so remove the 'nargs and action'

Reply:
when I tried to use this it wasn't working, in the video it looks like this was recorded before reordering the tabs, so I'm wondering if this broke while reordering.